challengeFive.js

```diff
@@ -0,0 +1,46 @@
+// First attempt 
+
+let cardNumber = "";
+
+function hideCardNumber(number){
+    const cardNumberString = number.toString();
+    const integers = cardNumberString.split("");
+    
+    for(let i = 0; i < integers.length - 4; i++) {
+        integers[i] = "*";
+        cardNumber += integers[i]
+    }
+   
+    for(let i = integers.length - 4; i < integers.length; i++){
+        cardNumber += integers[i]
+    }
```

Comment:
Not sure what you are trying to do with 2 for loops. I understand this works but seems very redundant.

You don't even need to append the integers to the cardNumber. You can just concatenate the integers array and return it.